mungegithub/github/github.go

```diff
@@ -1786,6 +1786,26 @@ func (obj *MungeObject) AddAssignee(owner string) error {
 	return nil
 }
 
+// IsAssignee will determine if the owner is a valid assignee in this repo
+func (obj *MungeObject) IsAssignee(owner string) (bool, error) {
```

Comment:
But I think you're right -- it doesn't require any special info other than what's in Config